### PR TITLE
Feature 14 Getting Seniority levels from the endpoint

### LINKED ISSUE #14: Feature - Getting Seniority levels from the endpoint
We need the seniority level to come from the API so we have a unified list and make easier the request from the API
## Checklist ✅

- [X] The issue is present in the latest release.
- [X] I have searched the [issues](https://github.com/Platzi-Master-C8/gethired-base/issues) of this repository and believe that this is not a duplicate.

## Summary 💡

The levels of seniority should come from an endpoint

## Requirements 🌈

The options of the seniority field should be fetched from the backend

## Motivation 🔦

When I was integrating the salaries chart with the API, noticed the request sent a seniority string but the endpoint expected a number. Instead of mapping the seniority levels from a string to a number on the client-side, we decided it would be better if we keep sending the string, but with the actual info fetched from the backend, so seniority levels can be updated from the backend without affecting any part of the front.

### PR DESCRIPTION
**Pull request by johannpereze**
## Checklist

- [X] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [X] Related issue has been created
- [X] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

### Type of PR

- [X] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Chore

### Merge strategy

- [X] Squash (default)
- [ ] Merge commit (branch merges, usually used for merges to master)
- [ ] Rebase (preserve individual commits without merge commit)

### Purpose

When I was integrating the salaries chart with the API, noticed the request sent a seniority string but the endpoint expected a number. Instead of mapping the seniority levels from a string to a number on the client-side, we decided it would be better if we keep sending the string, but with the actual info fetched from the backend, so seniority levels can be updated from the backend without affecting any part of the front.

### Relevant tickets

> Paste here the link to the task ticket
resolves #14 ] 

### Related PR's

> Paste here the links to related PR's
### Screenshots

image

> Add screenshots that could help others do understand even more the objective of your pr

### CODE REVIEW

File: src/components/FormCard/FormCard.jsx [hunk 1]
```diff
@@ -10,16 +10,27 @@ import { selectTechnologies, selectJobs } from 'App/ListData/selectors';
...
+}) => {
     const { title_id, technologies, seniority, english_level } = values;
-    const [ListSenority, setListSenority] = useState([]);
+    // const [ListSenority, setListSenority] = useState([]);
     const [ListEnglish, setListEnglish] = useState([]);
 
     useEffect(() => {
         addListData();
-        setListSenority(getListByName('senority'));
+        // setListSenority(getListByName('senority'));
```
Comment (yadurani) [inline src/components/FormCard/FormCard.jsx]: Please, remove this comment

File: src/components/FormCard/FormCard.jsx [hunk 2]
```diff
@@ -10,16 +10,27 @@ import { selectTechnologies, selectJobs } from 'App/ListData/selectors';
...
+    listTechnologies,
+    listJobs,
+    ListSenority,
+    children,
+    onDelete,
+    addListData,
+}) => {
     const { title_id, technologies, seniority, english_level } = values;
-    const [ListSenority, setListSenority] = useState([]);
+    // const [ListSenority, setListSenority] = useState([]);
```
Comment (yadurani) [inline src/components/FormCard/FormCard.jsx]: Please, remove this comment

File: src/components/FormCard/FormCard.jsx [hunk 3]
```diff
@@ -106,6 +118,7 @@ FormCard.defaultProps = {
 const mapStateToProps = (state) => ({
     listTechnologies: selectTechnologies(state),
     listJobs: selectJobs(state),
+    ListSenority: selectSeniority(state),
```
Comment (yadurani) [inline src/components/FormCard/FormCard.jsx]: please, put in lowercase first letter

**Review by yadurani — CHANGES_REQUESTED** [review]

File: src/components/FormCard/FormCard.jsx (hunk 2 above)
Comment (kevfarid) [inline src/components/FormCard/FormCard.jsx]: Quit this comment

File: src/components/FormCard/FormCard.jsx (hunk 1 above)
Comment (kevfarid) [inline src/components/FormCard/FormCard.jsx]: Same here. You should delete this comment.

**Review by kevfarid — CHANGES_REQUESTED** [review]

**Comment by kevfarid** [comment]
You must be adding title complete in the pr. Example: Feature [ID Issue] - [Name Issue]

**Comment by kevfarid** [comment]
You should add the **issue** in `Linked issues` that are located in the **sidebar**

**Review by kevfarid — DISMISSED** [review]
Nice

**Review by yadurani — DISMISSED** [review]

**Review by kevfarid — APPROVED** [review]

**Pull request merged**